src/ProjectFileTools.MSBuild/Worksapce.cs

```diff
@@ -48,17 +48,47 @@ internal Workspace(string filePath)
         /// <param name="sourceText">Text in the current file</param>
         /// <param name="position">Position of item that is to be resolved</param>
         /// <returns></returns>
-        public string ResolveDefinition(string filePath, string sourceText, int position)
+        public List<Definition> ResolveDefinition(string filePath, string sourceText, int position)
         {
             Verify.NotDisposed(this);
-            string file = null;
+            List<Definition> definitions = new List<Definition>();
 
             if (_project != null)
             {
                 XmlDocumentSyntax root = Parser.ParseText(sourceText);
                 SyntaxNode syntaxNode = SyntaxLocator.FindNode(root, position);
 
-                if (syntaxNode.ParentElement.Name.Equals(SyntaxNames.Import))
+                if ((int)syntaxNode.Kind == SyntaxNames.XmlLiteralTokenValue && Utilities.IsProperty(sourceText.Substring(syntaxNode.Span.Start, syntaxNode.FullWidth), position - syntaxNode.Span.Start, out string propertyName))
```

Comment:
nit: would be nice to split this method - (e.g. separate methods for now)
At minimum, some comments between the big pieces of logic would make it easier to follow